src/core/schema/common.json

```diff
@@ -102,6 +102,23 @@
           "type": "string"
         }
       }
+    },
+    "checkboxlist_data": {
+      "type": "object",
+      "properties": {
+        "label": {
+          "type": "string"
+        },
+        "value": {
+          "oneOf": [
```

Comment:
Let's keep the JSON as explicit as possible and use a `*_path` keyword of the value is  a jsonpath you have. It makes it easier for the front end to know if it should be doing a look-up or not.

Reply:
If you want to you could have the class take either and then come up with some way of distinguishing between it being a jsonpath or anything else and output the correct field.

Reply:
@PeterAlabaster: I can't resolve this because of the permissions feel free to do it :)

Reply:
Thanks man :)